lib/services/elasticsearch.js

```diff
@@ -631,43 +632,57 @@ class ElasticSearch extends Service {
     }
 
     esRequest.body = request.input.body.bulkData;
+    const kuzzleMetaCreated = {
+      author: userId,
+      createdAt: dateNow,
+      updatedAt: null,
+      updater: null,
+      active: true,
+      deletedAt: null
+    };
+    const kuzzleMetaUpdated = {
+      updater: userId,
+      updatedAt: dateNow
+    };
 
-    // set missing index & type if possible
-    esRequest.body.forEach(item => {
+    // set missing index & type if possible and add metadata
+    let lastAction;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the declaration of "lastAction" to line 652. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS2392)